src/SFA.DAS.AdminService.Web/Models/Error403ViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace SFA.DAS.AdminService.Web.Models
+{
+    public class Error403ViewModel
+    {
+        private readonly string _integrationUrlPart = string.Empty;
+
+        public Error403ViewModel(string environment)
+        {
+            if (!string.IsNullOrEmpty(environment) && !environment.Equals("prd", StringComparison.CurrentCultureIgnoreCase))
+            {
+                _integrationUrlPart = "test-";
+            }
+        }
+
+        /// <summary>
+        /// Gets DfESignIn Select service link.
+        /// </summary>
+        public string HelpPageLink => $"https://{_integrationUrlPart}services.signin.education.gov.uk/approvals/select-organisation?action=request-service";
```

Comment:
This looks really odd - why wouldn't this Url be a configuration property

Reply:
As the DfE SignIn Service url are most unlikely to get changed ever, we have hardcoded this Url in the Model rather than as the configuration,

Reply:
Changed the help link from configuration.